docs/src/main/asciidoc/security-openid-connect.adoc

```diff
@@ -693,22 +691,19 @@ import org.junit.jupiter.api.Test;
 import com.github.tomakehurst.wiremock.WireMockServer;
 import com.github.tomakehurst.wiremock.client.WireMock;
 
-import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
 import io.quarkus.test.oidc.server.OidcWireMock;
-import io.quarkus.test.oidc.server.OidcWiremockTestResource;
 import io.restassured.RestAssured;
 
 @QuarkusTest
-@QuarkusTestResource(OidcWiremockTestResource.class)
 public class CustomOidcWireMockStubTest {
 
     @OidcWireMock
     WireMockServer wireMockServer;
 
     @Test
     public void testInvalidBearerToken() {
-        wireMockServer.stubFor(WireMock.post("/realms/quarkus/protocol/openid-connect/token/introspect")
+        wireMockServer.stubFor(WireMock.post("/auth/realms/quarkus/protocol/openid-connect/token/introspect")
```

Comment:
Why do you have `/auth` added here - with Keycloak-X (used in the quickstart for testing) it is not supported by default. So I probably aligned the wiremock test code as well...

Reply:
@aona42 Thanks for verifying it all, I'll try to have a look asap at this specific point, in meantime can you please squash your commits and have the same commit message as this PR's name

Reply:
@aona42 I see, so the test lifecycle manager has `/auth` based stubs, the reason the test still passes because the introspection itself fails. So your fix is good. Please only squash the commits, thanks again for your thorough review of the guides

Reply:
@sberyozkin Done, thanks for reviewing.